docs/contracts/v2/guides/proxy-architecture/04-batch-stream.mdx

```diff
@@ -0,0 +1,297 @@
+---
+id: "batchstream"
+sidebar_position: 4
+title: "Batch stream"
+---
+
+import AdmonitionSimpleCode from "@site/docs/snippets/AdmonitionSimpleCode.mdx";
+
+In this guide, we will show you how to batch create streams via the Proxy and Sablier's Proxy Target programmatically.
+
+:::note
+
+This guide assumes that you have already read Create
+[Single Stream](/contracts/v2/guides/proxy-architecture/singlestream) section.
```

Comment:
Why does "Single Stream" matter here?

It has absolutely nothing to do with "Batch Stream".